testsuite/simulation/modelica/commonSubExp/wrapFunctionCalls12.mos

```diff
@@ -34,9 +34,9 @@ simulate(CSE.wrapFunctionCalls12); getErrorString();
 //
 // ########### Updated globalKnownVars (simulation) (3)
 // ========================================
-// 1: x:VARIABLE(unit = "s" fixed = true )  = sin(p1)  type: Real 
-// 2: p1:PARAM()  = 5.0  type: Real 
-// 3: y:VARIABLE(unit = "m" fixed = true )  = cos(x)  type: Real 
+// 1: x:VARIABLE(unit = "s" fixed = true )  = sin(p1)  type: Real
```

Comment:
That's weird: if `y` is made `PARAM`, why isn't `x`?
Is this done before matching-sorting?